src/main/resources/static/phoneClient.html

```diff
@@ -4,9 +4,12 @@
     <meta charset="UTF-8">
     <title>Phone Client</title>
     <script src="https://cdn.jsdelivr.net/npm/@stomp/stompjs@7.0.0/bundles/stomp.umd.min.js"></script>
+    <script src="https://d3js.org/d3.v7.min.js"></script>
     <script src="/phoneClientStompManager.js"></script>
 </head>
 <body>
+<svg width="250" height="250" id="svg"></svg>
```

Comment:
250 is 250px i guess maybe you want to make it more dynamic based on the size of the screen (browser window)

Reply:
"Minor comment"? Perhaps, in well-structured code showing a good understanding of d3. As it is... anyway, works now for phoneClient; the presentation will be able to choose the iframe dimensions too.